vtk.Dockerfile

```diff
@@ -0,0 +1,43 @@
+# syntax=docker/dockerfile:1
```

Comment:
Would prefer to move this file from the top level directory to the `operators/vtk_renderer` folder to which it relates and update reference docs accordingly.

Reply:
Looks like the file was copied rather than moved, this file can be deleted

Reply:
my bad, resolved in the last push